styles/_header.less

```diff
@@ -77,32 +78,30 @@
 .site-header__smallScreenNav {
   width: 150px;
   display: inline-block;
-  padding: 0px;
   text-align: left;
   margin: 10px 0 0 20px;
-  list-style-type:none
 }
 
 .site-header__smallScreenNav li {
   margin: 0 0 20px 0;
 }
 
 .site-header__smallScreenNav li a {
-  color: white;
+  color: @badgerWhite;
   text-decoration: none;
 }
 
 .site-header__menuCloseButton {
   position: absolute;
   top: 5px;
   left: 10px;
-  color: white;
+  color: @badgerWhite;
   text-decoration: none;
   cursor: pointer;
 }
 
 .site-header__smallScreenNavComponent {
-  width: 100%;
+  flex: 1;
```

Comment:
As discussed, vendor prefixes are only needed for browsers we are not supporting, so ignoring this for now.